File: shuffle/c/test_fisher_yates.c
#include "fisher_yates.h"

void test_fisher_yates()
{
	const int size = size;
	int array[size];
	int i;
	for(i = 0; i < size; i++)
		array[i] = i;
	fisher_yates(array, size);
}

void test_fisher_yates_front()
{
	const int size = size;
	int array[size];
	int i;
	for(i = 0; i < size; i++)
		array[i] = i;
	fisher_yates_front(array, size);
}

int main()
{
	test_fisher_yates();
	test_fisher_yates_front();
	return 0;
}

File: shuffle/py/TestFisherYates.py
import unittest
from fisher_yates import fisher_yates
from fisher_yates import fisher_yates_front

class TestFisherYates(unittest.TestCase):
    def testFisherYates(self):
        array = range(100)
        fisher_yates(array)

    def testFisherYatesFront(self):
        array = range(100)
        fisher_yates_front(array)

if __name__ == "__main__":
    unittest.main()
